ckanext/ogdchcommands/commands.py

```diff
@@ -79,10 +75,6 @@ def __init__(self, name):
             default=False,
             help='dryrun of cleanup harvestjobs and '
```

Comment:
This help text should either contain `cleanup_extras` or not list the commands that take the `--dryrun` option.